### PR TITLE
Welcome Page

### PR DESCRIPTION
**Pull request by LHJE**
# Welcome Page/Learn More Page

Go get the basic skeletal structure of the webpages. API calls are not made in this PR.

### ```GET / ``` - welcome

Displays the Welcome/landing page for our app. On this page we added: 
- a section describing why we are using Google

### ```GET / learn_more``` - learn_more

Displays a basic set up for the Learn More page, which should be filled out at a later date with information we as a group feel would be best to be there.

## Additional Notes

I've also updated the gems and application.scss so that navigation styling works correctly (hopefully).

Currently all tests are passing with 96.55% covered. The lines that aren't being hit are lines that will be hit when sensors, profile, and plants pages are made, and when OAuth has been implemented, to take care of the lines not being tested in the nav_spec and welcome_index_spec.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -13,5 +13,5 @@
  */
 
 // Custom bootstrap variables must be set or imported *before* bootstrap.
-// @import "bootstrap";
-// .solar-nav {float: right}
+@import "bootstrap";
```
Comment (dcoleman21) [inline app/assets/stylesheets/application.scss:16]: Good catch here! Now we can see all the pretty stuff we have done.

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -100,17 +97,24 @@ body {font-family: "Lato", sans-serif}
...
         <script src="https://apis.google.com/js/platform.js" async defer></script>
         <%= link_to "Login with Google", "/auth/google_oauth2", class: "w3-btn w3-blue-grey w3-block" %>
       </div>
 
-<!-- Visit the Learn More page button -->
-      <div class="w3-panel">
-        <%= link_to "Be The Change. Learn More", "/learn_more", class: "w3-btn w3-light-green w3-block" %>
+      <div class="w3-panel w3-pale-blue w3-block">
+        <h4 >Why we use Google for Login:</h4>
+        <p class="w3-justify w3-center "> - Google already has strong authorization, helping us protect your data - <br>
```
Comment (dcoleman21) [inline app/views/welcome/index.html.erb:113]: Thank you for adding this. Let's look into making that a partial and adding those CTA's that we spoke with Ian about. 
- Why use Google to login?
- set up a garden
- track your plants
- track your sensor data
- see your carbon impact
- see your Solar Garden community

**Review by dcoleman21 — CHANGES_REQUESTED** [review]
I have made some suggestions on here that we may want to implement. Let's chat about it before we merge.

File: app/views/welcome/index.html.erb (hunk 2 above)
Comment (LHJE) [reply app/views/welcome/index.html.erb:113]: Sounds good.  Let's take all this out and refactor it before merging

**Review by LHJE — COMMENTED** [review]
Looks like what I sent up!

**Review by dcoleman21 — APPROVED** [review]
Looks good

**Review by dcoleman21 — APPROVED** [review]
I love the changes

**Pull request merged**